doc/examples/plot_registration.py

```diff
@@ -59,7 +59,7 @@
 # --- Show the result
 
 fig = plt.figure(figsize=(15, 4))
-ax0, ax1, ax2 = fig.subplots(1, 3, True)
+ax0, ax1, ax2 = fig.subplots(1, 3)
```

Comment:
```suggestion
ax0, ax1, ax2 = fig.subplots(1, 3, sharex=True, sharey=True)
```
Probably not necessary, but it doesn't hurt ^^